install.sh

```diff
@@ -0,0 +1,266 @@
+
+## get and check the chosen script's option
+process_install="0"
+silent_install="0"
+if [ "$#" -eq 0 ] ; then
+    echo -e "Install : please select one option"
+    display_help
+    exit 1
+fi
+
+while getopts ":ih" options; do
+    case ${options} in
+        i ) silent_install="0"
+            process_install="1"
+            ;;
+        \?|h ) display_help;
+            exit 0
+            ;;
+        * )	display_help ;
+            exit 1
+            ;;
+    esac
+done
+
+if [ "$process_install" -ne 1 ]; then
+    echo "Install : option not found"
+    exit 1
+fi;
+
+## init LOG_FILE
+[ ! -d "$LOG_DIR" ] && mkdir -p "$LOG_DIR"
+if [ -e "$LOG_FILE" ] ; then
+	mv "$LOG_FILE" "$LOG_FILE.`date +%Y%m%d-%H%M%S`"
+fi
+${CAT} << __EOL__ > "$LOG_FILE"
+__EOL__
+
+## Init GREP,CAT,SED,CHMOD,CHOWN variables
+log "INFO" "Check mandatory binaries"
+define_specific_binary_vars
+
+## display header banner
+${CAT} << __EOT__
+
+###############################################################################
+#                                                                             #
+#                                                                             #
+#                        Centreon Gorgone daemon module                       #
+#                                                                             #
+#                                                                             #
+###############################################################################
+
+__EOT__
+
+## displaying the license
+echo -e "\nPlease read the license.\\n\\tPress enter to continue."
+read
+tput clear
+more "$BASE_DIR/LICENSE.txt"
+
+yes_no_default "Do you accept the license ?"
+if [ "$?" -ne 0 ] ; then
+    echo_info "As you did not accept the license, we cannot continue."
+    log "INFO" "Installation aborted - License not accepted"
+    exit 1
+else
+    log "INFO" "Accepted the license"
+fi
+
+## Test all binaries
+BINARIES="rm cp mv ${CHMOD} ${CHOWN} echo more mkdir find ${GREP} ${CAT} ${SED}"
+
+line="------------------------------------------------------------------------"
+
+echo "$line"
+echo -e "\tChecking all needed binaries"
+echo "$line"
+
+binary_fail="0"
+for binary in $BINARIES; do
+	if [ ! -e ${binary} ] ; then 
+		pathfind "$binary"
+		if [ "$?" -eq 0 ] ; then
+			echo_success "${binary}" "$ok"
+		else 
+			echo_failure "${binary}" "$fail"
+			log "ERROR" "\$binary not found in \$PATH"
+			binary_fail=1
+		fi
+	else
+		echo_success "${binary}" "$ok"
+	fi
+done
+
+## Script stop if one binary wasn't found
+if [ "$binary_fail" -eq 1 ] ; then
+	echo_failure "Please check failing binary and retry"
+	echo -e "\tThe logs are available in this file :\n$LOG_FILE"
+	exit 1
+fi
+
+echo -e "\n$line"
+echo -e "\tChecking the mandatory folders"
+echo -e "$line"
+
+## check filesystem space
+check_disk_space
+
+## define destination folders
+allow_creation_of_missing_folders
+locate_gorgone_logdir
+locate_gorgone_varlib
+locate_gorgone_etcdir
```

Comment:
Those directories need Gorgone's user and group rights.

Reply:
I'll find a way to do so